mistral/mixtral-8x7b-instruct-vllm/config.yaml

```diff
@@ -0,0 +1,15 @@
+base_image:
```

Comment:
curious why we need this base image?

Reply:
base image has `nvcc` and the right version of `torch` installed. There was a lot of messing around and this was the only configuration that didn't lead to a failed build of `megablocks`.